contracts/src/interfaces/IHyperdriveRegistry.sol

```diff
@@ -2,10 +2,58 @@
 pragma solidity 0.8.20;
 
 interface IHyperdriveRegistry {
-    /// @notice Allows anyone to get the info for a hyperdrive instance.
-    /// @param _hyperdriveInstance The hyperdrive instance address.
-    /// @return The uint256 value set by governance.
+    /// @dev The info related to each Hyperdrive instance.
+    struct HyperdriveInfo {
+        /// @dev Data about the instance. Different registries can utilize
+        ///      different schemas for these values.
+        uint256 data;
```

Comment:
From my perspective, `uint256` and `bytes32` offer the same room for creativity. They are both just 32 bytes. That said, I'm fine with switching this to `bytes32` since it would indicate that it's more of a "free form" field. 

I thought about changing it to `bytes` too. I like the idea, but I think we'll need a new function like `getHyperdriveDataSchema` that returns the ABI of the data field.